apache/superset
# fix(dependabot): correct pip and actions configs

### SUMMARY

This PR updates the pip and github-actions ecosystem configs for dependabot.  Previously these ecosystems would raise errors when dependabot attempted to run.  With these edits dependabot can successfully run.  It may be that the more correct change would be to remove these sections depending on the goals of the project, if that would be preferred I can certainly make that change.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TESTING INSTRUCTIONS


1. Fork the apache/superset repo
2. Enable Dependabot on the fork on the master branch (goto Insights->Dependency Graph->Dependabot and click "Enable")
3. Once Dependabot runs view logs for pip and actions to see errors
4. Merge this change to master on the new fork
5. Dependabot will run again, verify that the error for actions now refers to the open PR limit of 0. Pip may still error out due to a timeout, but it will open PRs successfully.  Once the python PRs are caught up it should no longer timeout.

### ADDITIONAL INFORMATION


- [x] Has associated issue: Fixes #21237
